packages/shared/src/utils/routes/routes.js

```diff
@@ -54,5 +54,30 @@ export const routes = {
     about_us: '/dashboard/about-us',
     explore: '/dashboard/explore',
     resources: '/dashboard/resources',
+
+    my_apps: '/dashboard/my-apps',
+
+    market_commodities: '/dashboard/markets/commodities',
+    market_forex: '/dashboard/markets/forex',
+    market_stock: '/dashboard/markets/stock',
+    market_synthetic: '/dashboard/markets/synthetic',
+    markets: '/dashboard/markets',
+
+    platform_binary_bot: '/dashboard/platforms/binary-bot',
+    platform_dbot: '/dashboard/platforms/dbot',
     platform_dmt5_synthetic: '/dashboard/platforms/dmt5_synthetic',
+    platform_dtrader: '/dashboard/platforms/dmt5_synthetic',
```

Comment:
typo?, also please add dmt5, dmt5-financial, dmt5-financial-stp, and change dmt5_synthetic to dmt5-synthetic